app/services/location_service/outcodes_that_straddle_the_m25.yaml

```diff
@@ -0,0 +1,52 @@
+# Derived from https://www.google.com/maps/d/viewer?mid=1_yZ8LNwwjFuMMKHlThA4khMQDARD-DYl&z=11&ll=51.493316716269405%2C0.12035871093750039
```

Comment:
Assuming these are a subset (m25 intersects) of the postcodes shown on that link, is the process of arriving at that subset documented?

Reply:
Adding a comment